mimic/test/test_maas.py

```diff
@@ -400,6 +432,282 @@ def test_delete_alarm(self):
         self.assertEquals(resp.code, 200)
         self.assertEquals(0, len(self.get_responsebody(resp)['values'][0]['alarms']))
 
+    def test_test_check(self):
+        """
+        The test-check API should return fake test-check results.
+        """
+        req = request(self, self.root, "POST",
```

Comment:
All these tests could use the `json_request` or `request`(in case on empty response body) from https://github.com/rackerlabs/mimic/blob/master/mimic/test/helpers.py and skip the three lines
```
req = request(self, self.root, "POST", self.uri + '/entities/' + self.entity_id + '/test-check', json.dumps({'type': 'agent.disk'}))
resp = self.successResultOf(req)
data = self.get_responsebody(resp)
```